.travis.yml

```diff
@@ -7,11 +7,15 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bower's cache
 
 env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=default
```

Comment:
Yes, I believe that is the case. I would be nice to keep it to five builds per our conversations about test suite speed. If we do sacrifice one for now, maybe it's `ember-canary`?